compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1538,88 +641,6 @@ bool OMR::Power::CodeGenerator::checkAndFetchRequestor(TR::Instruction *instr, T
    return false;
    }
 
-// PPC Codegen Peephole Pass
-// one forward pass over the code looking for a variety of peephole
-// opportunities
-void OMR::Power::CodeGenerator::doPeephole()
-   {
-   static char *disablePeephole = feGetEnv("TR_DisablePeephole");
-   if (disablePeephole)
-      return;
-
-   int syncPeepholeWindow = self()->comp()->isOptServer() ? 12 : 6;
-
-   if (self()->comp()->getOptLevel() == noOpt)
-      return;
-
-   TR::Instruction *instructionCursor = self()->getFirstInstruction();
-
-   while (instructionCursor)
-      {
-      self()->setCurrentBlockIndex(instructionCursor->getBlockIndex());
```

Comment:
Do we do the same in the new peephole pass?

Reply:
No, because there is no peephole which consumes this. In fact these APIs should be deprecated as they are currently unused. I've opened #5293 to track this.